tests/data_context/cloud_data_context/test_datasource_crud.py

```diff
@@ -0,0 +1,135 @@
+"""This file is meant for integration tests related to datasource CRUD."""
+import copy
+from unittest.mock import patch
+
+import pytest
+
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.store import GeCloudStoreBackend
+from great_expectations.data_context.types.base import DatasourceConfig
+from great_expectations.datasource import BaseDatasource
+
+
+@pytest.mark.integration
```

Comment:
Should we also mark with `cloud`? I've been doing that for tests that we want run each and every CI run (see `dgest-cloud-overrides` section of YAML).

Reply:
Done!